<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TerrainProfileDialog</class>
 <widget class="QDialog" name="TerrainProfileDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ì§€í˜• ë‹¨ë©´ í”„ë¡œíŒŒì¼ëŸ¬ (Terrain Profiler) - ArchToolkit</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_Header">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;ì§€í˜• ë‹¨ë©´ ë¶„ì„&lt;/span&gt;&lt;/p&gt;&lt;p&gt;ì§€ë„ìƒì˜ ê°€ìƒ ì ˆë‹¨ì„ ì„ ë”°ë¼ ê³ ë„ ë³€í™”ë¥¼ ê·¸ë˜í”„ë¡œ ì‹œê°í™”í•©ë‹ˆë‹¤.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="styleSheet">
      <string>background-color: #fff3e0; padding: 10px; border-radius: 4px; border: 1px solid #ffe0b2;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupDem">
     <property name="title">
      <string>1. ëŒ€ìƒ DEM ì„ íƒ</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QgsMapLayerComboBox" name="cmbDemLayer"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupSettings">
     <property name="title">
      <string>2. ë‹¨ë©´ ì¶”ì¶œ ë„êµ¬</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="lblSamples">
        <property name="text">
         <string>ìƒ˜í”Œë§ ì •ë°€ë„ (ì  ê°œìˆ˜):</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinSamples">
        <property name="maximum">
         <number>5000</number>
        </property>
        <property name="value">
         <number>200</number>
        </property>
        <property name="singleStep">
         <number>50</number>
        </property>
        <property name="toolTip">
         <string>ë‹¨ë©´ì„  ìƒì—ì„œ ê³ ë„ë¥¼ ì¶”ì¶œí•  ì§€ì ì˜ ê°œìˆ˜ì…ë‹ˆë‹¤. ë†’ì„ìˆ˜ë¡ ì •ë°€í•©ë‹ˆë‹¤.</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnDrawLine">
        <property name="text">
         <string>âœï¸ ì§€ë„ì— ë‹¨ë©´ì„  ê·¸ë¦¬ê¸°</string>
        </property>
        <property name="styleSheet">
         <string>font-weight: bold; background-color: #ff9800; color: white; padding: 5px 10px;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnClear">
        <property name="text">
         <string>ì´ˆê¸°í™”</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupProfile">
     <property name="title">
      <string>3. ë‹¨ë©´ ê²°ê³¼ ê·¸ë˜í”„</string>
     </property>
     <layout class="QVBoxLayout" name="chartLayout">
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item>
       <widget class="QLabel" name="lblStats">
        <property name="text">
         <string>ì§€ë„ì— ì„ ì„ ê·¸ë¦¬ë©´ ì—¬ê¸°ì— ë‹¨ë©´ ê·¸ë˜í”„ì™€ í†µê³„ê°€ í‘œì‹œë©ë‹ˆë‹¤.</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="styleSheet">
         <string>color: #999; font-style: italic;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="btnExportCsv">
       <property name="text">
        <string>ğŸ’¾ CSV ë°ì´í„° ì €ì¥</string>
       </property>
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnExportImage">
       <property name="text">
        <string>ğŸ–¼ï¸ ì´ë¯¸ì§€ ì €ì¥</string>
       </property>
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnClose">
       <property name="text">
        <string>ë‹«ê¸°</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsMapLayerComboBox</class>
   <extends>QComboBox</extends>
   <header>qgis.gui</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
